Runtime/Scripts/LocationService.cs

```diff
@@ -99,27 +96,39 @@ public static IEnumerator InitalizeLocationService(int maxWait = 20, bool contin
 
         /// <summary>
         /// EnsureLocation Confirm that user location is valid, user location is essential for MobiledgeX services
-        /// If Location permission is denied by User an exception will be thrown (for Android) in LocationServiceFlow()
-        ///                                                                      (for iOS) in InitalizeLocationService()
+        /// If Location permission is denied by User an exception will be thrown once RetrieveLocation() is called
         /// </summary>
         public static IEnumerator EnsureLocation()
```

Comment:
Can we add some logs in this function so that devs know why they can't proceed with our APIs?